programs/system/src/lib.rs

```diff
@@ -30,14 +30,19 @@ pub mod light_system_program {
     };
     use super::*;
 
-    pub fn init_cpi_context_account(ctx: Context<InitializeCpiContextAccount>) -> Result<()> {
+    pub fn init_cpi_context_account(_ctx: Context<InitializeCpiContextAccount>) -> Result<()> {
+        unimplemented!("CPI context is not enabled");
```

Comment:
Shouldn't this be guarded by `#[cfg(not(feature = "cpi_context"))]`?

Reply:
we tried that. It didn't seem to work in this place not sure why.